config/samples/ramendr_v1alpha1_drpolicy.yaml

```diff
@@ -3,6 +3,12 @@ kind: DRPolicy
 metadata:
   name: drpolicy-sample
 spec:
+  schedule: "0 * * * *"  # hourly
+  replicationClassSelector:
+    matchLabels:
+      # appclass: gold
```

Comment:
(nit) drop these comments from the sample as well. Again samples get processed into the bundle, so we need it clean. `class: ramen` is a nice label choice.

Reply:
ACK